tests/functional/adapter/incremental/test_incremental_on_schema_change.py

```diff
@@ -82,19 +82,23 @@ class TestIncrementalOnSchemaChange(BaseIncrementalOnSchemaChange):
         select 1 as id,
         cast('2020-01-01 01:00:00' as datetime) as date_hour,
         1 as field_1,
-        2 as field_2 union all
+        2 as field_2,
+        STRUCT(1 as `group`) union all
```

Comment:
I wonder if it still works when the columns within the STRUCT are a mix of quoted and un-quoted identifiers with a mix of uppercase and lowercase?

Something like this:
```sql
STRUCT(1 as `group`, 2 as `WHERE`, 3 as group_2, 4 as WHERE_TO) union all
```